### DISCUSSION TITLE
Can't get output?

### DISCUSSION THREAD

**Discussion by kayakaan02** [szpajder/dumphfdl#30] [Q&A] [answered]
I'm trying to read from a pre-recorded audio file I/Q raw
So i use something like 
`dumphfdl --output decoded:text:file:path=/some/path/a.log --iq-file "/some/path/b.wav" --sample-rate 48000 --sample-format CF32 --centerfreq 0 1.416`
the sample rate/format/center freq etc is random here because i dont actually remember
It creates a.log but it's empty
and in terminal, it gives me this: 
```
dumphfdl 1.3.0 (libacars 2.1.4-9ed8fb0)
Waiting for all threads to finish
Shutting down decoder thread
output_file(/some/path/a.log): shutting down
```
So no output in terminal nothing in a.log, I tried so much samples and iq-file options so it's not the case about input i think
Would appreciate if you can help me

**Comment by szpajder** [comment]
> the sample rate/format/center freq etc is random here because i dont
> actually remember
>
In this case please record a new file, with known parameters and try once
again.

If it still does not work, please share how you actually record the file
(ie. what program do you use and how do you invoke it).

**Reply by kayakaan02** [reply]
> the sample rate/format/center freq etc is random here because i dont actually remember
> In this case please record a new file, with known parameters and try once again. If it still does not work, please share how you actually record the file (ie. what program do you use and how do you invoke it).

I got the sample from sigidwiki, in hfdl page.(https://www.sigidwiki.com/wiki/High_Frequency_Data_Link_(HFDL))
It is labeled as "Audio Sample" at the right side.
https://www.sigidwiki.com/images/4/4c/HFDL_Sound.mp3

So I know the sample rate(44100), center frequency(1450) and sample format(CF32)
Here's the code:
`dumphfdl --output decoded:text:file:path=/path/asd.txt --iq-file "/path/HFDL_Sound.wav" --sample-rate 44100 --sample-format CF32 --centerfreq 1450 1451.0`
(I converted mp3 to wav with audacity)

Results are the same

edit: maybe the .wav file im using is not supported? so if it is how can i use dumphfdl with a pre-recorded file?

**Comment by szpajder** [comment, marked as answer]
This is audio recording, not I/Q. dumphfdl does not deal with audio signals.

**Reply by kayakaan02** [reply]
Sorry for disturbing again but, do you think can I use dumphfdl with that file if it's converted to an I/Q file?

**Comment by szpajder** [comment]
>
> Sorry for disturbing again but, do you think can I use dumphfdl if I
> convert that .wav file to an iq file?
>
If you can, then go for it. However it would probably be easier and quicker
to just record a new file. Or if you want to decode that particular file,
then just use a decoder which reads audio (eg. PC-HFDL).
